modules/configuring-default-seccomp-profile.adoc

```diff
@@ -1,15 +1,99 @@
 [id="configuring-default-seccomp-profile_{context}"]
-= Configuring the default seccomp profile
-OpenShift ships with a default seccomp profile that is referenced as `runtime/default`. You can enable the default seccomp profile for a pod or container workload by setting `RuntimeDefault` as following:
+= Enabling the default seccomp profile for all {product-title} pods
+{product-title} ships with a default seccomp profile that is referenced as `runtime/default`. In {product-version} newly created pods have the Security Context Constraint (SCC) set to `restricted-v2` and the default seccomp profile applies to the pod. 
 
-.Example
 
-[source, yaml]
+. You can verify Security Context Constraint (SCC) set on a pod by running the following commands: 
+
+.. Verify what pods are running in the namespace: 
++
+[source, terminal]
+----
+$ oc get pods -n <namespace> 
+---- 
++
+For example to verify what pods are running in the `workshop` namespace run the following: 
++
+[source, terminal]
+----
+$ oc get pods -n workshop 
+---- 
++
+.Example output 
++
+[source, terminal]
+----
+NAME                READY   STATUS      RESTARTS   AGE
+parksmap-1-4xkwf    1/1     Running     0          2m17s
+parksmap-1-deploy   0/1     Completed   0          2m22s
+---- 
++
+.. Inspect the pods: 
++
+[source, terminal]
+----
+$ oc get pod parksmap-1-4xkwf -n workshop -o yaml
+----
++
+.Example output 
++
+[source, terminal]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    k8s.v1.cni.cncf.io/network-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    k8s.v1.cni.cncf.io/networks-status: |-
+      [{
+          "name": "openshift-sdn",
+          "interface": "eth0",
+          "ips": [
+              "10.131.0.18"
+          ],
+          "default": true,
+          "dns": {}
+      }]
+    openshift.io/deployment-config.latest-version: "1"
+    openshift.io/deployment-config.name: parksmap
+    openshift.io/deployment.name: parksmap-1
+    openshift.io/generated-by: OpenShiftWebConsole
+    openshift.io/scc: restricted-v2 <1>
+    seccomp.security.alpha.kubernetes.io/pod: runtime/default <2>
+---- 
+<1> The `restricted-v2` is set by default on newly created pods in {product-version}. 
+<2> The `restricted-v2` sets the seccompProfile to `runtime/default` by default. 
+
+You can enable the default seccomp profile for a pod or container workload that does not have the seccompProfile set by adding `restricted-v2` to the default service account. 
```

Comment:
This is incorrect. All authenticated users (this includes ServiceAccounts) have access to the restricted-v2 SCC.

Reply:
So @mvazquezc if I have a pod that was running on 4.10 and I upgraded to 4.11 it will have seccompProfile set to `runtime/default` by default is that correct. I tried to do a few tests. I maybe missing something subtle here and maybe I am showing my lack of knowledge but "All authenticated users (this includes ServiceAccounts) have access to the restricted-v2 SCC" does that mean the same as all running pods will have seccompProfile set to `runtime/default` by default. Here I am trying to address the situation where a pod may not have it set but maybe this is not a valid case now in 4.11.

Reply:
Well, there is a subtle difference indeed. 

If a 4.11 cluster was upgraded from 4.10, the default SCC will continue to be `restricted`. 

If it's a 4.11 fresh install, the default SCC will be `restricted-v2`, and pods will get the default seccomp profile configured.